docs/examples/color_channels/rgb_grayscale.jl

```diff
@@ -0,0 +1,22 @@
+# ---
+# title: RGB to GrayScale
+# cover: assets/rgb_grayscale.png
+# ---
+
+# This example illustrates RGB to Grayscale Conversion
+
+
+using Images, TestImages
```

Comment:
```suggestion
using ImageCore, TestImages
```

Sorry that I missed this.